### DISCUSSION TITLE
7.3.0

### DISCUSSION THREAD

**Discussion by arthanson** [netbox-community/pynetbox#599] [Announcements]
* Add support for NetBox v3.7 by @markkuleinio #596
* Updated test suite by @abhi1693 

<hr /><em>This discussion was created from the release <a href='https://github.com/netbox-community/pynetbox/releases/tag/v7.3.0'>7.3.0</a>.</em>

**Comment by jcollie** [comment]
7.3.0 hasn't shown up on PyPI yet...

**Reply by arthanson** [reply]
Yes, there was an issue deploying to pypi, will debug and fix tomorrow.

**Reply by markkuleinio** [reply]
Do you @arthanson know what's the problem that usually happens that the docker image for NetBox 3.3 does not always work (it tries to create duplicate directory or something like that)? You hit it in your last setup.py change last night when fixing the publishing, and I hit it in #601. Ended up just removing 3.3 from tests as the other versions (3.4, 3.5, 3.6, 3.7) work fine.
